### PR TITLE
add condition code constant for REMARK, or always true condition

### PR DESCRIPTION
**Pull request by arvidn**

### CODE REVIEW

**Comment by arvidn** [comment]
this was implemented here: https://github.com/Chia-Network/chia_rs/pull/4

**Review by prozacchiwawa — APPROVED** [review]

**Review by richardkiss — APPROVED** [review]

File: chia/wallet/puzzles/condition_codes.clvm [hunk 1]
```diff
@@ -35,4 +35,7 @@
     ; block index
     (defconstant ASSERT_HEIGHT_RELATIVE 82)
     (defconstant ASSERT_HEIGHT_ABSOLUTE 83)
+
+    ; A condition that is always true and always ignore all arguments
+    (defconstant REMARK 1)
```
Comment (richardkiss) [inline chia/wallet/puzzles/condition_codes.clvm:40]: How about `REM`? 😜

Comment (arvidn) [reply chia/wallet/puzzles/condition_codes.clvm:40]: I'm fine either way. what do other people think? @prozacchiwawa @matt-o-how ?

**Review by wjblanke — APPROVED** [review]
aok

**Pull request merged**